packages/client-sdk-nodejs/src/config/middleware/experimental-metrics-logging-middleware.ts

```diff
@@ -42,6 +99,11 @@ class ExperimentalMetricsLoggingMiddlewareRequestHandler extends ExperimentalMet
  * your {Configuration} to enable this middleware.
  */
 export class ExperimentalMetricsLoggingMiddleware extends ExperimentalMetricsMiddleware {
+  private static readonly metricsLogInterval = 1000;
+  private static readonly eventLoopDelayInterval = 20;
+  private static eldMonitor: EventLoopDelayMonitor;
+  private static elu: EventLoopUtilization;
+  private static isLoggingStarted = false;
```

Comment:
i'm not seeing where you set this to `true`

Reply:
Ha. I guess my plan to only run it once would have failed.